mongodbatlas/organizations.go

```diff
@@ -188,3 +212,138 @@ func (s *OrganizationsServiceOp) Delete(ctx context.Context, orgID string) (*Res
 
 	return resp, err
 }
+
+// ListUnacceptedInvitations gets all unaccepted invitations to the specified Atlas organization.
+//
+// See more: https://docs.atlas.mongodb.com/reference/api/organization-get-invitations/
+func (s *OrganizationsServiceOp) ListUnacceptedInvitations(ctx context.Context, orgID string, opts *InvitationOptions) (*[]Invitation, *Response, error) {
```

Comment:
[nit] let's move all invitation code to a new file, to be clear, don't change `OrganizationsService` it should stay the same but you can move the invitation implementations to a new file `organization_invitations(_test).go` or something